src/server/controller/user/removeInvitation.ts

```diff
@@ -0,0 +1,44 @@
+import { CountryIso } from '@meta/area'
+import { ActivityLogMessage, Assessment, Cycle } from '@meta/assessment'
+import { RoleName, User, UserRole } from '@meta/user'
+
+import { BaseProtocol, DB } from '@server/db'
+import { ActivityLogRepository } from '@server/repository/assessment/activityLog'
+import { UserRepository } from '@server/repository/public/user'
+import { UserRoleRepository } from '@server/repository/public/userRole'
+
+export const removeInvitation = async (
+  props: {
+    countryIso: CountryIso
+    assessment: Assessment
+    cycle: Cycle
+    invitationUuid: string
+    user: User
+  },
+  client: BaseProtocol = DB
+): Promise<UserRole<RoleName>> => {
+  const { countryIso, assessment, cycle, invitationUuid, user } = props
+
+  return client.tx(async (t) => {
+    const userRole = await UserRoleRepository.remove({ invitationUuid }, t)
+
+    const invitedUser = await UserRepository.getOne({ id: userRole.userId })
+
+    await ActivityLogRepository.insertActivityLog(
+      {
+        activityLog: {
+          target: { user: invitedUser.name, role: userRole.role },
```

Comment:
target should be: {userId:invitedUser.id, role...}

Reply:
in the old fra_audit were stored that way...

https://user-images.githubusercontent.com/10047945/191969097-3e4ee1bd-f860-4a84-9af6-693515b992b3.mov

Reply:
I can add it but not in all the migrated entries.